### PR TITLE
Feature/update prod value

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: src/chainbridgeConfig.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export const chainbridgeConfig: ChainbridgeConfig = {
       ],
     },
     {
-      chainId: 1,
+      chainId: +((process.env.REACT_APP_CERE_CHAIN_ID as unknown) as number),
```
Comment (AndreiNavoichyk) [inline src/chainbridgeConfig.ts]: Why did you change it?

File: .env.prod [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+REACT_APP_ETHEREUM_NETWORK_NAME="Ethereum Mainnet"
+REACT_APP_ETHEREUM_BRIDGE_ADDRESS="0xCaF65AB2eC9B39403966991eb34B1e8B9E44C041"
+REACT_APP_ETHEREUM_ERC20_HANDLER_ADDRESS="0x8fe028Eb002bbc3ec45c5dF8acfFf67eC95B6f88"
+REACT_APP_ETHEREUM_RPC_URL=""
+REACT_APP_ETHEREUM_ERC20_TOKEN_ADDRESS="0x2da719db753dfa10a62e140f436e1d67f2ddb0d6"
+REACT_APP_ETHEREUM_ERC20_NAME="CERE Network"
+REACT_APP_ETHEREUM_ERC20_SYMBOL="CERE"
+
+### Cerebellum Network
+REACT_APP_CERE_CHAIN_ID=11
```
Comment (AndreiNavoichyk) [inline .env.prod]: Why `11`?

File: src/chainbridgeConfig.ts (hunk 1 above)
Comment (karthikeyan-cere) [reply src/chainbridgeConfig.ts]: Because we need to pass different value for Testnet and Mainnet

File: .env.prod (hunk 2 above)
Comment (karthikeyan-cere) [reply .env.prod]: As we discussed in call, I kept it as 11

Comment (AndreiNavoichyk) [reply .env.prod]: We discussed that you will try this locally to check our theory but later you will research the official documentation about chain ids generation.

**Review by AndreiNavoichyk — APPROVED** [review]
